src/Footer.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Footer = props => {
+  const addresses = props.address;
+  const mainAddress = addresses.map((info, index) => (
+    <li key={index}>{info}</li>
+  ));
+  return <ul className="footer">{mainAddress}</ul>;
```

Comment:
A very minor point, but you could make this a bit shorter:

```js
const mainAddress = props.addresses.map((info, index) => (
  <li key={index}>{info}</li>
));
return <ul className="footer">{mainAddress}</ul>;
```

And you could make that even shorter:

```js
return (
  <ul className="footer">
    {props.addresses.map((info, index) => (
      <li key={index}>{info}</li>
    ))}
  </ul>
);
```